.github/workflows/deploy.yml

```diff
@@ -0,0 +1,105 @@
+name: DEPLOY
+
+on:
+  push:
+    branches: [ '**' ]
+    tags-ignore: [ '**' ]
+
+jobs:
+
+  build:
+    if: contains(github.event.head_commit.message, 'deploy+')
+    strategy:
+      matrix:
+        container: [ "ubuntu:18.04", "ubuntu:19.04", "ubuntu:20.04", "debian:9", "debian:10", "fedora:29", "fedora:30", "fedora:31", "archlinux:latest" ]
+    runs-on: "ubuntu-latest"
+    container:
+      image: ${{ matrix.container }}
+      env:
+        DIST: ${{ matrix.container }}
+    steps:
+      - name: checkout-source
+        uses: actions/checkout@v1
+        with:
+          submodules: recursive
+      - name: run-builder
+        shell: bash
+        run: ./tools/build.sh
+      - name: upload-artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: package-archives
+          path: artifact
+
+  deploy:
+    needs: build
+    runs-on: "ubuntu-latest"
+    container:
+      image: "archlinux:latest"
+    env:
+      BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
+      BINTRAY_APIKEY: ${{ secrets.BINTRAY_APIKEY }}
+      BINTRAY_GPG_DATA: ${{ secrets.GPG_DATA }}
+      BINTRAY_GPG_ID: ${{ secrets.GPG_ID }}
+      JFROG_CLI_OFFER_CONFIG: false
+    steps:
+      - name: checkout-source
+        uses: actions/checkout@v1
+        with:
+          submodules: recursive
+      - name: install-jfrog-cli
+        uses: jfrog/setup-jfrog-cli@v1
+      - name: download-artifacts
+        uses: actions/download-artifact@v1
+        with:
+          name: package-archives
+          path: artifact
+      - name: bintray-publish
+        shell: bash
+        run: ./tools/publish.sh
+
+  release:
+    needs: deploy
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: checkout-source
+        uses: actions/checkout@v1
+      - name: get-version
+        id: get_version
+        shell: bash
+        run: |
+          RELEASE_VERSION=$(cat version.txt | head -n1)
+          echo ::set-output name=RELEASE_VERSION::$RELEASE_VERSION
+      - name: download-artifacts
+        uses: actions/download-artifact@v1
+        with:
+          name: package-archives
+          path: artifact
+      - name: create-release-bundle
+        shell: bash
+        run: |
+          zip -j9 installers.zip artifact/*
+      - name: create-release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think we'll need this additional secret also, aren't we?

Reply:
No this is automatically generated by github for a workflow run.

[authenticating-with-the-github_token](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret):
> GitHub automatically creates a `GITHUB_TOKEN` secret to use in your workflow. You can use the `GITHUB_TOKEN` to authenticate in a workflow run.